tests/results/test_clearable_file_field.html

```diff
@@ -0,0 +1,26 @@
+<form  method="post"  enctype="multipart/form-data">
+    <div id="div_id_clearable_file" class="mb-3">
+        <label for="id_clearable_file" class="form-label">Clearable file</label>
+        <div class=" mb-2">
+            <div class="input-group mb-2">
+                <span class="input-group-text">Currently</span>
+                <div class="form-control d-flex h-auto">
+                    <span class="text-break" style="flex-grow:1;min-width:0">
+                        <a href="something"></a>
+                    </span>
+                    <span class="align-self-center ml-2">
+                        <span class="form-check">
+                            <input type="checkbox" name="clearable_file-clear" id="clearable_file-clear_id" class="form-check-input" >
+                            <label class="form-check-label mb-0" for="clearable_file-clear_id">Clear</label>
+                        </span>
+                    </span>
+                </div>
+            </div>
+            <div class="input-group mb-0 ">
+                <input type="file" name="clearable_file" class="form-control" id="id_clearable_file">
+            </div>
+            <div class="input-group mb-0">
```

Comment:
This line here looks odd, I don't think think we need it? I'm not sure where it is coming from? 🤔

Reply:
Fixed, it was a remnant of the double help text.